layers/+lang/clojure/packages.el

```diff
@@ -286,6 +286,19 @@
         (kbd "t")   'cider-test-run-test
         (kbd "T")   'cider-test-run-ns-tests)
 
+      (evilified-state-evilify-map cider-repl-history-mode-map
+        :mode cider-repl-history-mode
+        :bindings
+        "j" 'cider-repl-history-forward
+        "k" 'cider-repl-history-previous
+        "s" (if (featurep 'swiper)
+                'swiper
+              'helm-swoop)
```

Comment:
How frequent do you think that occurs? But even in that case, I would prefer to suggest in this way to the user to use the equivalent of `helm-swoop` (I guess a user that does not have swiper or helm-swoop available, will be knowledgeable enough to know how to fix it).

Reply:
It's not likely to occur. But if that happens this becomes an error when pressing the key bindings.

Reply:
I would not mind that error, because then users must take action (if they want to use the keybinding). Now, I fixed it but the users without ivy/helm will get lead to use occur, which I would dissuade.

Anyway, I also understand that you prefer to have no errors, but I would prefer users get the error because I find it better that they get it (or maybe invent some other mechanism to remind them that they use inferior commands).

I am not sure how many users without helm and ivy (or even with) will use this history functionality because until now nobody seemed to miss any cider-repl-history keybindings at all...

Reply:
I prefer explicit error, not 'no error'. That is, should user have neither helm nor ivy, either don't add that binding, or emit an more informative error message right away.

Reply:
`evilified-state-evilify-map` is a function so you can use apply/funcall.

Just make a list of bindings. Then if user use ivy/helm, add one extra entry. And finally apply the list to the function.

Reply:
Sorry, I think I forgot about this. I could still add it, but I really think this is an extremely rare case. And otherwise, a very small issue, and quick to fix issue. So probably let's leave it for now. And again, thanks for your work!